user_tools/README.md

```diff
@@ -17,7 +17,7 @@ The wrapper improves end-user experience within the following dimensions:
 
 ## Getting started
 
-Set up a Python environment with a version between 3.8 and 3.10
+Set up a Python environment with a version between 3.8 and 3.11
```

Comment:
I thought we supported till 3.12?

Reply:
We did at some point.
Then some dependencies made new releases which either work [3.8-3.11], or [3.9-3.12]

Reply:
Makes sense. Thanks @amahussein.